include/aspect/material_model/rheology/strain_dependent.h

```diff
@@ -54,6 +54,17 @@ namespace aspect
         plastic_weakening_with_plastic_strain_and_viscous_weakening_with_viscous_strain,
         viscous_weakening_with_viscous_strain_only
       };
+      /**
+       * Enumeration for selecting which type of healing mechanism to use.
+       * For none, no strain healing occurs.
+       * Otherwise, the material is healed based on a temperature dependent
```

Comment:
```suggestion
       * Otherwise, the strain is healed (reduced) as a function of temperature,
       * a user defined healing time scale and additional parameters.
```